vendingMachine.js

```diff
@@ -13,34 +13,50 @@ let stock = [
 function showVendingMachine(arr1, arr2) {
   let result = '';
   let coord, stoc;
+  let space = " ";
+  let supline = "-";
+  let supli = supline.repeat(122);
+  console.log(supli);
   for (let i = 0; i < arr1.length; i++) { //0
     for (let j = 0; j < arr1[i].length; j++) { //2
-      result = result + arr1[i][j] + '    ';
+      let a = 0;

```

Comment:
use a better name, the variables name must be a descriptive name that explains the value that it contains

Reply:
Done